src/args.ts

```diff
@@ -29,8 +29,7 @@ function getParams() {
       },
       env: {
         alias: 'e',
-        describe: 'Override the mappings in config with process.env environment variables.',
-        type: 'string',
```

Comment:
`type: 'string'` is contradictory to `boolean: true` so went ahead and removed it.